### ISSUE TITLE
Moving 'Events' from Sources to the top main menu

### ISSUE THREAD

**Issue by VickyGarnett** [DARIAH-ERIC/dariah-campus#1000]
### Bug report

Issue 1000 - woohoo!

But in all seriousness - can we create a link to 'Events' in the top main menu, and move it out of 'Sources'?  It no longer makes sense to keep it there, I don't think.

**Comment by stefanprobst** [comment]
events are removed from "sources" - do we still want to add them to the top-level navigation? (similar question probably for "pathfinders")

**Comment by stefanprobst** [comment]
btw, new navigation entries can be added to the site via cms: https://campus.dariah.eu/keystatic/branch/main/singleton/en%3Anavigation

**Comment by VickyGarnett** [comment]
Hi @stefanprobst - again, this one fell foul of my leg break, apologies.

I think, if memory serves (@ttasovac please correct me if I'm wrong) there was a bit of discussion around whether we actually wanted to separate 'Events' out from the 'Resources' page, because I think creating its own separate menu item would mean doing that, am I right?  And if that's the case, I think we came the conclusion that it was better to have events in amongst other resources than on a different page.

However, we are going to look at the design of the events pages again, so if it ends up looking quite different, then maybe it would make sense to revisit this discussion at that stage?

**Comment by stefanprobst** [comment]
https://campus.dariah.eu/resources/events does exist - if you want that link in the top navigation bar, you can add it via cms: https://campus.dariah.eu/keystatic/branch/main/singleton/en%3Anavigation

**Comment by ttasovac** [comment]
Indeed. Since events are no longer part of resources, to me it makes sense to add them in the menu. @VickyGarnett please go ahead.

**Comment by stefanprobst** [comment]
> Since events are no longer part of resources

they *are* part of resources. we have 4 types of resources: hosted, external, pathfinders, and events.

events and pathfinders have additional pages which only list events / pathfinders because we have the "Browse contents" cards for them on the landing page.

**Comment by ttasovac** [comment]
Ah, sorry. I misread the above where you said they were removed from _sources_. Ok in that case, I'll close this. I agree with @VickyGarnett we don't need a separate menu item right now, and in case we decide to have it (if we ever do some kind of reshuffling inside our not-ideal data model) we can revisit this...
